components/CustomError.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Paper, Alert, Button } from '@mui/material'
+
+const CustomError = ({ title, message, buttonMessage }) => {
+  return (
+    <div style={{ textAlign: 'center' }}>
+      <Paper elevation={ 4 } sx={{ p: 1, backgroundColor: '#FFD700' }}>
```

Comment:
can we use color names instead of codes?

Reply:
Are there any preferences for the color of the title, or is it ok if the background remains white?